docs/development/upgrading-node.md

```diff
@@ -70,49 +66,35 @@ We need to generate a patch file from each patch applied to V8.
 4. Inspect Node [repo](https://github.com/electron/node) to see what patches upstream Node
   used with their v8 after bumping its version
   - `git log --oneline "deps/v8"`
+  - The commit message "update V8 to <version>" represents replacing `deps/v8` with the named version of V8, exactly as it is in upstream. The commit message "patch V8 to <version>" isn't a complete replacement of `deps/v8`, but just applying the changes in that version.
 5. Create a checklist of the patches. This is useful for tracking your work and for
   having a quick reference of commit hashes to use in the `git diff-tree` step below.
-6. Read `patches/common/v8/README.md` to see which patchfiles came from the previous version of V8 and therefore need to be removed.
-  - Delete each patchfile referenced in `patches/common/v8/README.md`
-7. Apply all patches with the [`get-patch` script](https://github.com/electron/libchromiumcontent/blob/master/script/README.md#get-patch):
+6. Apply all patches with the [`get-patch` script](https://github.com/electron/electron/blob/master/script/README.md#get-patch):
   - `./script/get-patch --repo src/v8 --output-dir patches/v8 --commit abc123 def456 ...`
-8. Update `patches/common/v8/README.md` with references to all new patches that have been added so that the next person will know which need to be removed.
-9. Update Electron's submodule references:
-  ```sh
-  $ cd electron/vendor/node
-  electron/vendor/node$ git fetch
-  electron/vendor/node$ git checkout electron-node-vA.B.C
-  electron/vendor/node$ cd ../libchromiumcontent
-  electron/vendor/libchromiumcontent$ git fetch
-  electron/vendor/libchromiumcontent$ git checkout upgrade-to-chromium-X
-  electron/vendor/libchromiumcontent$ cd ../..
-  electron$ git add vendor
-  electron$ git commit -m "update submodule references for node and libcc"
-  electron$ git push origin upgrade-to-chromium-<VERSION>
-  electron$ script/bootstrap.py -d
-  electron$ script/build.py -c -D
-  ```
+7. Update `patches/common/v8/README.md` with references to all new patches that have been added so that the next person will know which need to be removed.
+8. Update Electron's [DEPS](https://github.com/electron/electron/blob/master/DEPS#L5) with the commit hash of the latest commit in the `electron-node-vX.Y.Z` branch.
 
 ## Notes
 
-- libcc and V8 are treated as a single unit
 - Node maintains its own fork of V8
   - They backport a small amount of things as needed
   - Documentation in node about how [they work with V8](https://nodejs.org/api/v8.html)
 - We update code such that we only use one copy of V8 across all of electron
-  - E.g electron, libcc, and node
+  - E.g electron, chromium, and node
```

Comment:
capital letter for "chromium"?

Reply:
Not new to this PR, but `Electron` and `Node.js` as well